examples/private-data/src/main.rs

```diff
@@ -0,0 +1,36 @@
+#[macro_use]
+extern crate rocket;
+
+use rocket::{Config, State};
+use rocket::fairing::AdHoc;
+
+#[cfg(test)] mod tests;
+
+#[get("/encrypt/<msg>")]
+fn encrypt_endpoint(msg: &str, config: &State<Config>) -> String{
+    let secret_key = config.secret_key.clone();
+    let encrypted = secret_key.encrypt(msg).unwrap();
+
+    info!("received message for encrypt: '{}'", msg);
+    info!("encrypted msg: '{}'", encrypted);
+
+    encrypted
+}
+
+#[get("/decrypt/<msg>")]
+    fn decrypt_endpoint(msg: &str, config: &State<Config>) -> String {
+        let secret_key = config.secret_key.clone();
+        let decrypted = secret_key.decrypt(msg).unwrap();
+
+        info!("received message for decrypt: '{}'", msg);
+        info!("decrypted msg: '{}'", decrypted);
+
```

Comment:
Not sure what's going on with the formatting here.

Reply:
For some reason, the formatting of the code stopped working for me; neither cargo fmt nor autoformat when saving a file in VSCode is functioning. 
I'll fix it.

Reply:
I believe we have intentionally disabled it, since it doesn't work properly for some of the more complex types and where clauses.